src/PublicProfile/PublicProfile.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import Header from '../Header';
+import Footer from '../Footer';
+import ProfileImage from './profileImage';
+import ProfileDetails from './profileDetails';
+import SkillLabel from './skillLabel';
+import Projects from '../Projects';
+import './PublicProfile.css';
+
+class PublicProfile extends React.Component {

```

Comment:
public profile will need a corresponding type file.

call it like,

class PublicProfile extends React.Component<PublicProfileProps, {}> 
the empty object represents state, which is currently empty for you. 

you need to make a PublicProfileProps interface in the Redux.d file, take a look at the other examples. 

you will probably need the following to start
user
project
getProjects